tests/test_datadog.py

```diff
@@ -42,3 +44,55 @@ def test_api_key():
     configured_api_key = DataDog.settings[DataDog.KEY_DATADOG_API_KEY]
 
     assert configured_api_key == 'test_api_key'
+
+
+def test_metrics():
```

Comment:
So... this for me falls into the area where I believe we need to be careful about how we do PR review. I try very hard to word all my PR comments as *questions* or (at strongest) *suggestions*. For example, _would it be better to split this test method into one per function_?. The primary purpose of PR review is to *spot things that may go wrong, not to try and force someone else to follow the reviewer's *preferences*.

In this case, splitting up the test function means that the setup has to be repeated across functions, but I can see that's the way you've structured it, so I'll follow along.